backend/index.js

```diff
@@ -1,96 +1,12 @@
-const fs = require('fs');
-const readline = require('readline');
-const {google} = require('googleapis');
+const express = require('express');
 
-// If modifying these scopes, delete token.json.
-const SCOPES = ['https://www.googleapis.com/auth/calendar.readonly'];
-// The file token.json stores the user's access and refresh tokens, and is
-// created automatically when the authorization flow completes for the first
-// time.
-const TOKEN_PATH = 'token.json';
+const app = express();
+app.use(express.json({ extended: true }))
```

Comment:
According to expree.js API there is not extended property in express.json options https://expressjs.com/ru/api.html

Reply:
Fixed.